build.sbt

```
@@ -1,83 +1,16 @@
-name := "circe-config"
-description := "Yet another Typesafe Config decoder"
-homepage := Some(url("https://github.com/circe/circe-config"))
-licenses += "Apache-2.0" -> url("https://www.apache.org/licenses/LICENSE-2.0.html")
-apiURL := Some(url("https://circe.github.io/circe-config/"))
-
-ThisBuild / organization := "io.circe"
-ThisBuild / crossScalaVersions := List("2.12.15", "2.13.8")
-ThisBuild / scalaVersion := crossScalaVersions.value.last
-
-ThisBuild / githubWorkflowJavaVersions := Seq("adopt@1.8")
-ThisBuild / githubWorkflowPublishTargetBranches := Nil
-ThisBuild / githubWorkflowBuild := Seq(
-  WorkflowStep.Sbt(
-    List(
-      "clean",
-      "coverage",
-      "scalafmtCheckAll",
-      "scalafmtSbtCheck",
-      "test",
-      "coverageReport"
-    ),
-    id = None,
-    name = Some("Test")
-  ),
-  WorkflowStep.Use(
-    UseRef.Public(
-      "codecov",
-      "codecov-action",
-      "v1"
-    )
-  )
+ThisBuild / tlBaseVersion := "0.8"
+ThisBuild / startYear := Some(2017)
+ThisBuild / organizationName := "Jonas Fonseca"
```

```suggestion
ThisBuild / organizationName := "Circe"
```

This would probably be good to change to Circe.

---

I think sbt-circe-org just sets this to `circe` but that overwrites the existing license headers on `sbt headerCreate`. That's the reason why I overwrote it here.

---

If we're not ok to change the copyright headers, there's probably a workaround.

---

Right, you can override the `headerLicense` setting.
https://github.com/typelevel/cats-effect/blob/5f7b24395d8a1df9ddbb5440025ba85c21ce8854/project/Common.scala#L34-L36

---

I tried `headerLicense` before and scoped it with `ThisBuild` but that had no effect. Putting the setting in the project worked. Thanks!

---

@zmccoy and I discussed, and we think the right thing to do is to set the `organizationName` to `Circe` and NOT override the existing headers, by adding double attribution headers.   You can do this by adding a blank block comment to each of the files with existing headers and re-running the header creation.  See this[ http4s file](https://github.com/http4s/http4s/blob/0e253e30bf983545cb0ebdd6407495b629f4ed1e/core/shared/src/main/scala/org/http4s/ResponseCookie.scala#L22) for an example.

In addition to the double headers, we think adding a notices file like [this one ](https://github.com/http4s/http4s/blob/0e253e30bf983545cb0ebdd6407495b629f4ed1e/NOTICE) that references the original licensed work [here](https://github.com/jonas/circe-config) would be best.

---

That's a good suggestion. Done in 15a73d1 and 993f07a.